src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -1137,11 +1137,15 @@ Value ByteCodeInterpreter::interpret(ExecutionState& state, ByteCodeBlock* byteC
                 :
             {
                 IteratorStep* code = (IteratorStep*)programCounter;
-                Value iterator = registerFile[code->m_iterRegisterIndex];
-                Value nextResult = iteratorStep(state, iterator);
+                Value nextResult = iteratorStep(state, registerFile[code->m_iterRegisterIndex]);
 
                 if (nextResult.isFalse()) {
```

Comment:
State expression explicitly
if (nextResult.isFalse() == true)

Reply:
Fixed. Thank you for the review.